python/tank/authentication/ui/recent_box.py

```diff
@@ -21,6 +21,7 @@
 from .qt_abstraction import QtGui
 from .qt5_like_line_edit import Qt5LikeLineEdit
 from .completion_filter_proxy import CompletionFilterProxy
+from tank_vendor.shotgun_api3.lib.six.moves import range
```

Comment:
Why range?

Reply:
As per comment above, modernize automatically did this and I wasn't looking for it (I don't think we had many range calls in python-api).  It's because of range/xrange changes but in our uses it is not necessary.